src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -111,7 +118,12 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   }
 
   async componentDidMount() {
-    const { store, dependentStores, isClusterScoped } = this.props;
+    const { store, dependentStores, isClusterScoped, isConfigurable, tableId } = this.props;
+    if (isConfigurable && tableId) {
+      const filters = get(userStore.preferences.tablesConfiguration, tableId);
+      if (filters != undefined) this.columnFilter = filters;
+      else this.columnFilter = ( 2 ** this.props.renderTableHeader.length ) - 1;
```

Comment:
this.columnFilter = filters ?? (( 2 ** this.props.renderTableHeader.length ) - 1);